MinecraftClient/Resources/config/MinecraftClient.ini

```diff
@@ -11,6 +11,7 @@ password=
 serverip=
 type=mojang                        # Account type. mojang or microsoft
 method=mcc                         # Microsoft Account sign-in method. mcc OR browser
+mcleaks=false
```

Comment:
Maybe you could allow `type=mcleaks` in addition to `mojang` and `microsoft` instead of adding another setting.

Reply:
Done!

Reply:
Oh wait haha I did it in the method instead of the type gimme a sec

Reply:
There that should work now